tests/ed25519_tests.rs

```diff
@@ -46,6 +46,14 @@ fn test_signature_ed25519() {
         let public_key = test_case.consume_bytes("PUB");
         assert_eq!(32, public_key.len());
 
+        // Test deterministic creation of key pair from seed.
+        let mut seed_bytes = [0;32];
```

Comment:
Small nit: what do you think about moving this chunk into its own test function?

Reply:
Thank you a lot for taking a look, and for your comments!

To me, this seemed a natural place to add this test case with few modifications to existing code, since this is the place where the public key is read and used, and where both the seed and the public key are already available. The existing function already tests many different features that all depend on the read values and can therefore be naturally tested as a logical unit in this function.  Would there be any benefit in adding a new function just for testing the relation between seed and public key? I will do it if that is necessary to merge this pull&nbsp;request.

Reply:
> Would there be any benefit in adding a new function just for testing the relation between seed and public key?

Main benefit would be isolation of functionality, but is outweighed by the points you brought up.

> I will do it if that is necessary to merge this pull request.

I'm just a contributor with no say over merge-ability.